servicetest/dbus/test_dbus.py

```diff
@@ -23,7 +23,42 @@
 import subprocess
 import dbusapp
 
-from testframework import ContainerApp
+from testframework import Container
+
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+# This function is used by the 'agent' fixture to know where the log should be stored
+def logfile_path():
+    return CURRENT_DIR + "/test.log"
+
+
+# The path to where the test app is located, will be passed in the 'data' dict when
+# starting the container. The test app is assumed to be located in the
+# same directory as this test.
+HOST_PATH = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Can't you add this directly in the DATA section? Or is it used somewhere else also?